.github/paths-labeller.yml

```diff
@@ -21,3 +21,5 @@
     - "x-pack/plugins/synthetics/**/*.*"
     - "x-pack/plugins/ux/**/*.*"
     - "x-pack/plugins/observability/public/components/shared/exploratory_view/**/*.*"
+  - "Team:obs-ux-management":
```

Comment:
Is it about adding a team label to the related PR?

Reply:
yeah, this will auto add label